src/jit/ee_il_dll.cpp

```diff
@@ -163,6 +163,20 @@ CorJitResult CILJit::compileMethod (
         return g_realJitCompiler->compileMethod(compHnd, methodInfo, flags, entryAddress, nativeSizeOfCode);
     }
 
+    if (!JitConfig.isInitialized())
+    {
+        JitConfig.initialize(compHnd, ProcessHeapAllocator::Singleton());
+
+        /* Initialize the single instance of the norls_allocator (with a page
+         * preallocated) which we try to reuse for all non-simulataneous
+         * uses (which is always, for the standalone). This depends on config
+         * values in debug builds and therefore must be performed after said
+         * values have been initialized.
+         */
+
+        nraInitTheAllocator();
```

Comment:
I don't understand why this needs to be here, instead of as the very first thing in compStartup(), where it was before. Now there could be lots of code run before the allocator is initialized. That seems dangerous.